src/inverseRobot.js

```diff
@@ -4,10 +4,30 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const values = Object.values(robot);
+
+  const newRobot = {};
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
```

Comment:
You can iterate the keys of robot using [in operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) to avoid nested loops